src/client/src/components/SortableSelectionList/index.tsx

```diff
@@ -33,6 +35,7 @@ const SortableSidebarItem = SortableElement(
         handleInputChange={handleInputChange}
         handleCloseClick={handleCloseClick}
         idx={idx + 1}
+        totalPageCount = {totalPageCount}
```

Comment:
for consistency, should we have `totalPageCount={totalPageCount}` (removing the spaces)